### PR TITLE
Switch from Travis CI to GitHub Actions

### LINKED ISSUE #15: Migrate CI from Travis to GitHub Actions
Travis as a service has become somewhat unreliable. The facility to keep running our CI tests with Travis is unknown, at the moment we have not as TNA hit their new "free" limit... but many projects I work on have. Whilst I personally like Travis, I think for the stability of the project we should likely move to GitHub Actions.

Comments?

### PR DESCRIPTION
**Pull request by adamretter**
Closes https://github.com/nationalarchives/kettle-jena-plugins/issues/15

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+name: CI
```
Comment (langsamu) [inline .github/workflows/ci.yml:1]: `mvn verify` [fails with license violation in this file](https://github.com/langsamu/kettle-jena-plugins/runs/1872469749?check_suite_focus=true#step:5:2847).

Will Maven recognise a license in YAML comments?
Or ignore the file?

Comment (adamretter) [reply .github/workflows/ci.yml:1]: This was already fixed in https://github.com/nationalarchives/kettle-jena-plugins/pull/24 which you merged. I can rebase this PR if you want to see the CI pass first. One moment...

Comment (adamretter) [reply .github/workflows/ci.yml:1]: Okay, I rebased and fixed the GitHub config license bit.

Comment (langsamu) [reply .github/workflows/ci.yml:1]: #24 did not fix license violation but b4ccd4a does.

**Pull request merged**